helm-v3-user-stories.md

```diff
@@ -0,0 +1,39 @@
+# Helm v3 User Stories
+
+This documented lists [user stories](https://en.wikipedia.org/wiki/User_story) for the development of Helm's 3rd major version. The user stories here are not a complete set for Helm but those that build upon Helm 2. The stories are grouped by role as defined in the [User profiles](user-profiles.md).
+
+## 1. Application Operator
+
+1. As an application operator, I want to be able to pull charts from within the cluster so that I can manage multiple clusters without exposing details to the CI system
+1. As an application operator, I want to be able to pull charts, so that my dev team can bootstrap local envs easily (e.g. w/Minikube or Docker)
+1. In order to have a secure system as an application operator, I want Helm to install/upgrade/delete based on the user’s credentials
+1. In order to have a secure system as an application operator, I want to be able to see who ran a particular install or other operation
+1. In order to have a secure system as an application operator, I want out-of-the-box “easy security”
+1. In order to have a secure system as an application operator, I want RBACs done for me
+1. In order to have a secure system as an application operator, I want to audit what charts have been deployed
+1. In order to have a secure system as an application operator, I want a pull model so that I can manage multiple private cluster from one authoritative source
+1. In order to have a secure system as an application operator, I want to enable RBAC on an in-cluster controller so that I can prevent external changes from happening
+1. In order to easily get started as an application operator, I want to easily install and use Helm
```

Comment:
Possible additions:
1. As an application developer, I want to have an easy way of accessing all logs of an install/upgrade so that I can easily troubleshoot or monitor application installs/upgrades 
2. As an application developer, I want to be able to specify dependencies on other installed releases so that I can install more complex systems